intelmq/bots/parsers/json/parser.py

```diff
@@ -12,15 +12,19 @@
 from intelmq.lib.bot import ParserBot
 from intelmq.lib.message import MessageFactory
 from intelmq.lib.utils import base64_decode
+import json
 
 
 class JSONParserBot(ParserBot):
     """Parse IntelMQ-JSON data"""
     splitlines = False
+    multiple_events = False
 
     def process(self):
         report = self.receive_message()
-        if self.splitlines:
+        if self.multiple_events:
+            lines = [json.dumps(event) for event in json.loads(base64_decode(report['raw']))]
```

Comment:
Converting the data forth and back appears to be inefficient.

Reply:
Yeah, could imagine this. Any tips to do this a proper way?

Currently this PR is running in our production and works just fine, but I agree on the double JSON conversion isn't the most efficient way to do this.